tests/test_formats__audio.py

```diff
@@ -162,8 +163,13 @@ def test_call(self):
 
         assert self.quux("~basename")
         assert self.quux("~dirname") == os.path.dirname(self.quux("~filename"))
-        assert self.quux("title") == \
-            "%s [Unknown]" % fsn2text(self.quux("~basename"))
+
+        basename = fsn2text(self.quux("~basename"))
+        stem = stem_of_file_name(basename)
+        extension = extension_of_file_name(basename)[1:]
+        assert self.quux("title") == config.gettext(
```

Comment:
I think we should _set_ these config items in the test and explicitly assert against expected values (otherwise it's just testing the implementation again slightly)